hive.yaml

```diff
@@ -0,0 +1,100 @@
+package:
+  name: hive
+  version: 4.0.0
+  epoch: 0
+  description: Apache Hive
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash
+      - openjdk-8-default-jvm
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - maven
+      - openjdk-8
+  environment:
+    LANG: en_US.UTF-8
+    JAVA_HOME: /usr/lib/jvm/java-1.8-openjdk
+    HIVE_VERSION: 4.0.0
+    HADOOP_VERSION: 3.3.6
```

Comment:
you can replace this environment variable `HADOOP_VERSION` with a melange variable.

```yaml
vars:
   hadoop-version: 3.3.6
```

then you use `${{vars.hadoop-version}}` and you don't have to repeat yourself in the runtime environment below (if they are intended to be the same thing).

Reply:
Love it! Thank you @smoser.